shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaReport.scala

```diff
@@ -152,20 +181,36 @@ trait ShivaReportTrait extends MultisampleMappingReportTrait {
 
   /** Single sample page */
   override def samplePage(sampleId: Int, args: Map[String, Any]): Future[ReportPage] = Future {
-    val variantcallingSection =
-      if (variantcallingExecuted)
-        List(
-          "SNV Calling" -> ReportSection(
-            "/nl/lumc/sasc/biopet/pipelines/shiva/sampleVariants.ssp"))
-      else Nil
-    val svSection =
-      if (svCallingExecuted)
-        List(
-          "SV Calling" -> ReportSection(
-            "/nl/lumc/sasc/biopet/pipelines/shiva/sampleVariantsSv.ssp"))
-      else Nil
+    var addedSections: List[(String, ReportSection)] = List()
```

Comment:
but it is needed, the operator ':+' that I use, doesn't modify the existing list but creates a new one